coral/src/app/features/approvals/acls/components/AclApprovalsTable.test.tsx

```diff
@@ -0,0 +1,263 @@
+    expect(within(header).getAllByRole("columnheader")[4]).toHaveTextContent(
+      "Team"
+    );
+    expect(within(firstRow).getAllByRole("cell")[4]).toHaveTextContent("Ospo");
+  });
+
+  it("has column to decsribe the acl type", () => {
+    expect(within(header).getAllByRole("columnheader")[5]).toHaveTextContent(
+      "ACL type"
+    );
+    expect(within(firstRow).getAllByRole("cell")[5]).toHaveTextContent(
+      "CONSUMER"
+    );
+  });
+
+  it("has column to decsribe the status", () => {
+    expect(within(header).getAllByRole("columnheader")[6]).toHaveTextContent(
+      "Status"
+    );
+    expect(within(firstRow).getAllByRole("cell")[6]).toHaveTextContent(
+      "Created"
+    );
+  });
+
+  it("has column to decsribe the author of the request", () => {
+    expect(within(header).getAllByRole("columnheader")[7]).toHaveTextContent(
+      "Requested by"
+    );
+    expect(within(firstRow).getAllByRole("cell")[7]).toHaveTextContent(
+      "amathieu"
+    );
+  });
+
+  it("has column to decsribe the timestamp when the request was made", () => {
+    expect(within(header).getAllByRole("columnheader")[8]).toHaveTextContent(
+      "Requested on"
+    );
+    expect(within(firstRow).getAllByRole("cell")[8]).toHaveTextContent(
+      "06-Jan-2023 14:50:37 UTC"
+    );
+  });
+
+  it("has column for action to view request details", async () => {
+    await userEvent.click(
+      within(within(firstRow).getAllByRole("cell")[9]).getByRole("button", {
+        name: "View topic request for aivtopic1",
+      })
+    );
+    expect(onDetails).toHaveBeenNthCalledWith(1, aclRequests[0]);
+  });
+
+  it("has column for action to approve request", async () => {
+    await userEvent.click(
+      within(within(firstRow).getAllByRole("cell")[10]).getByRole("button", {
+        name: "Approve schema request for aivtopic1",
+      })
+    );
+    expect(onApprove).toHaveBeenNthCalledWith(1, aclRequests[0]);
+  });
+
+  it("has column for action to decline request", async () => {
+    await userEvent.click(
+      within(within(firstRow).getAllByRole("cell")[11]).getByRole("button", {
+        name: "Decline topic request for aivtopic1",
+      })
+    );
+    expect(onDecline).toHaveBeenNthCalledWith(1, aclRequests[0]);
+  });
+
+  it("shows one header row and two data rows", () => {
+    const rows = screen.getAllByRole("row");
+    expect(rows).toHaveLength(3);
+    expect(rows[0]).toHaveTextContent("Principals/Usernames");
+    expect(rows[1]).toHaveTextContent("mbasani");
+    expect(rows[2]).toHaveTextContent("User:*");
+  });
+
+  it("renders (prefixed) in Topic cell when appropriate", () => {
+    const prefixedCells = screen.getAllByText("(prefixed)");
+    const notPrefixedCells = screen.getAllByText("aivtopic1");
+
+    expect(prefixedCells).toHaveLength(1);
+    expect(notPrefixedCells).toHaveLength(1);
+  });
+
+  it("renders all values for cells who can have multiple values", () => {
```

Comment:
(I know you moved that test over, to feel free to ignore)

I think the test could be more readable if it was more clear that `"mbasani maulbach "` is actually two values. I tripped over that right now 🙈  and had to check out the code and mocked values to make sure.

Moving them to variables like: 

```
const userOne = "mbasani"
const userTwo = "maulbach" 
```

and then using ``${userOne} ${userTwo} `` as value for "textContent" could make that more clear. 

it's a really tiny nitpick tho 😬